code/game/atoms.dm

```diff
@@ -708,64 +790,76 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 		blood_overlay.color = color
 		overlays += blood_overlay
 
-/atom/proc/clean_blood()
+/atom/proc/clean_blood(radiation_clean = FALSE)
```

Comment:
Yes, this really is the best proc, sadly.

We effectively made `washed` into `clean_blood`.

I wasn't going to port yet ANOTHER huge refactor with cleaning just to get this to work--this already touched an obscene amount of stuff.